blog/_posts/2017-09-12-Final_blog_post.md

```diff
@@ -0,0 +1,44 @@
+---
+title: “”
+layout: post
+created_at: Tue Sept 12 2017
+permalink: blog/2017-09-06-final-blog-post-team-victorious-secret
```

Comment:
Permalink and file name doesn't match. Please fix the date and name both. Fix a date everywhere with Wed Sept 13 2017.